sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/models.py

```diff
@@ -527,50 +850,32 @@ class CertificatePolicy(object):
     # pylint:disable=too-many-instance-attributes
     def __init__(
         self,
+        x509_properties,  # type: X509Properties
+        issuer_parameters,  # type: IssuerParameters
         attributes=None,  # type: Optional[models.CertificateAttributes]
         cert_policy_id=None,  # type: Optional[str]
         key_properties=None,  # type: Optional[KeyProperties]
         content_type=None,  # type: Optional[models.SecretContentType] or str
```

Comment:
`Optional[models.SecretContentType] or str` is an invalid annotation. I think what you want here is `Optional[Union[models.SecretContentType, str]]` which indicates a `SecretContentType`, string, or `None` is expected.

Reply:
sounds good, thanks!